src/index.ts

```diff
@@ -6,6 +6,7 @@ import { startLegacyNotarize, waitForLegacyNotarize } from './legacy';
 import { isNotaryToolAvailable, notarizeAndWaitForNotaryTool } from './notarytool';
 import { stapleApp } from './staple';
 import { NotarizeOptions, NotaryToolStartOptions } from './types';
+import { checkSignatures } from './check-signature';
```

Comment:
These are currently alpha sorted, could we put this at the top of this block to retain that?